.prettierignore

```diff
@@ -1,2 +1,3 @@
 __generated__
 src/desktop/components/react/stitch_components/index.tsx
+./hokusai/*.yml
```

Comment:
Hmmm since this is an autogenerated file, I don't think this is good to have here...

Reply:
Unfortunately putting `# prettier-ignore` in the Yaml files doesn't seem to work for me